frontend/src/app/organizations/organizations-list/organizations-list.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="row">
+  <div class="col-lg-12">
+    <h2>Lista organizacji</h2>
+  </div>
+</div>
+<div class="row">
+  <div class="col-lg-12">
+    <ng-container *ngIf="organizations">
+      <table *ngIf="organizations.length; else noOrganizations" class="table table-striped">
+        <thead>
+        <tr>
+          <th scope="col">Nazwa</th>
+          <th scope="col">Adres</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr *ngFor="let organization of organizations">
+          <td><a [routerLink]="[organization.slug, organization.id]">{{ organization.name }}</a></td>
```

Comment:
I also will no work that way outside of `/organizations` path (assuming that we want to add this in `/me` it will try to navigate to `/me/organization-slug/1`

Reply:
Now it should work correctly.